### ISSUE TITLE
"undeclared variable" error when using superenum

### ISSUE THREAD

**Issue by kindlychung** [julia-vscode/julia-vscode#787]
![afbeelding](https://user-images.githubusercontent.com/995761/60766239-450d6880-a0a7-11e9-9f35-5dc6ffce1d95.png)

**Comment by davidanthoff** [comment]
I believe the problem here is that we generally don't have a strategy for handling identifiers in macros. @ZacLN, is that right? We'll probably tackle this in some future version.

**Comment by ZacLN** [comment]
Yep there's a brief discussion of this attached to an earlier issue: https://github.com/julia-vscode/julia-vscode/issues/657

**Comment by davidanthoff** [comment]
The language server backend was rewritten in v1.220.0: StaticLint.jl and SymbolServer.jl were folded into JuliaWorkspaces, and completion, hover, signature help, go-to-definition and find-references were reimplemented. This report predates that rewrite, so the code that produced it no longer exists and the behaviour cannot be confirmed against the current engine.

Closing as outdated. If you still see this on a current release, please open a fresh issue with a reproducer and we will look at it against the new engine.
